Extensions/DialogueTree/tests/assignment.json

```diff
@@ -0,0 +1,81 @@
+[
```

Comment:
Thanks for including these tests :) How we can we run them? 
Could we include game examples using them?

Reply:
yes of course, game example will follow after this and yarn are ready :)